docs/tutorials/fundamentals/viewer.md

```diff
@@ -93,61 +72,80 @@ nbscreenshot(viewer, alt_text="photograph of an astronaut in napari viewer")
 
 ```{code-cell} python
 :tags: [remove-cell]
-
 viewer.close()
 ```
 
-Both the {func}`imshow<napari.imshow>` and the {meth}`add_image<napari.components.viewer_model.ViewerModel.add_image>`
-methods accept any numpy-array like
-object as an input, including n-dimensional arrays. For more information on
-adding images to the viewer see the [image layer guide](../../howtos/layers/image).
-Now we will continue exploring the rest of the viewer.
+ {func}`imshow<napari.imshow>` and the {meth}`add_image<napari.components.ViewerModel.add_image>` methods accept any numpy-array like object as input, including n-dimensional arrays. For more information on adding images to the viewer see the [image layer guide](../../howtos/layers/image). 
+
+Now we will continue exploring the rest of the viewer.  
 
 +++
 
 ## Layout of the viewer
 
-The viewer is organized into a few key areas:
+The viewer is organized into a few key areas which are explained in the next sections:
+* Main Menu (top bar menu)
+* Layer Controls
+* Layer Buttons
+* Layer List
+* Viewer Buttons
+* Status Bar
+* Canvas
+* Dimension Sliders
+* Scroll Buttons
+* Frame Playback  
+
+The image below has the areas of the viewer labeled: 
 
-![image: viewer layout](../assets/tutorials/viewer_layout.jpg)
+  ![image: viewer layout](../assets/tutorials/Viewer-with-arrows.png)
 
 We'll go through each of these in the next sections.
 
-+++
+### Main menu or top bar menu
```

Comment:
I still don't like these obvious screenshots with no extra info (or even alt text).

Reply:
Peter, the goal is to flesh them out later.

Reply:
I think +1 here, if we're providing details further down, we should present these screenshots there. I think the top bar menu is too standard a feature to highlight its structure to this degree. Definitely understand the importance of explaining the different napari-specific options, but I think this section should be removed.

Reply:
The idea is to fill in more information on these menus at a later date.  I'll talk to Dannielle and see what she thinks.

Reply:
I removed the screenshots and summarized each of the top-bar menus.  If this is still too much (and Dannielle is agreeable) I will delete the section on the top-bar menu.